Case study for Cloud Native EdgeComputing Satellite (ENG) (#655)
kubeedge/website: src/pages/case-studies/satellite/index.mdx

and software designed for specific payloads. One satellite, one mission, one launch defining its entire lifecycle. 
This closed system architecture necessarily leads to hardware and software incompatibility, inability to interchange and reuse, 
and very slow technological iteration.

To address these issues, satellite manufacturers both domestically and internationally proposed software-defined satellites from a software-defined perspective. 
Software-defined satellites are a new type of intelligent satellite centered on space-based computing, adopting an open system architecture that supports plug-and-play payloads and on-demand application loading. 
They can conveniently redefine satellite functions through software updates, flexibly adapting to multiple tasks and user types. Examples include SmartSat, GSky-1, Eutelsat Quantum, and OneSat, etc.

However, software-defined satellites still face numerous difficulties in software or application packaging, distribution, deployment, upgrades, and maintenance. 
They struggle to meet the satellite's on-orbit business requirements for rapid iteration, instant deployment, grayscale release, and continuous evolution. 

> As a result, both domestic and international efforts are trying to develop a new generation of intelligent satellites.

Reviewer: ```suggestion
As a result, the industry is trying to develop a new generation of intelligent satellites.
```

configuration files, and backend services required to build applications are integrated, achieving self-management.

Traditional communication and remote sensing satellites only have storage capabilities, merely storing data to be transmitted back to the ground, without onboard processing capabilities.
By utilizing cloud-native edge computing technology, satellites can process data in space, significantly reducing transmission pressure and meeting the diverse needs of aerospace fields. 

**Significance: Giving Satellites "Apps", Enabling "Business in Space and Services in Orbit"**

As shown in the diagram, by reconstructing the KubeEdge system and lightweight AI inference model Sedna according to satellite characteristics, empowering satellites with cloud-native edge computing capabilities, and achieving collaborative AI inference between satellites and ground stations, the following was accomplished:
- In remote sensing scenarios, on-orbit target identification accuracy improved by over 50%
- Satellite data transmission volume reduced by 90%
- Support for in-orbit continuous AI model updates

> ![Satellite](images/satellite.png)

Reviewer: Do you have an editable version of this image? Maybe we can also replace the Chinese in the picture with English

Author: I had this english one so added : ) also Elias, a lot of Indian devs were interested in these case studies so I will create english versions for other case studies as well, please review them as well : )

Reviewer: Sounds great! Just go ahead!
